temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package temporal.api.cloud.namespace.v1;
+
+option go_package = "go.temporal.io/api/cloud/namespace/v1;namespace";
+
+import "google/protobuf/timestamp.proto";
+
+message CertificateFilterSpec {
```

Comment:
pls add comment describing the use of certificate filters

Reply:
@jlacefie there is a comment in the namespace spec object where this struct is used.